lib/controllers/admin.ts

```diff
@@ -523,5 +523,22 @@ export default (crowi: Crowi) => {
     return res.json(ApiResponse.success({ message: 'Now re-building backlinks ... this takes a while.' }))
   }
 
+  actions.api.user.edit = async function (req: Request, res: Response) {
+    const { name, emailToBeChanged, id } = req.form.userEditForm
+
+    try {
+      const user = await User.findById(id)
+      if (!user) throw new Error('User not found')
+      const emailDuplicateCheck = await User.findUserByEmail(emailToBeChanged)
+      if (emailDuplicateCheck) throw new Error('Email duplicated')
```

Comment:
```suggestion
      if (emailDuplicateCheck) {
        throw new Error('Email duplicated')
      }
```

他の箇所もそうなんだけど、既存コードもそうなってるとおもうけどこっちのほうがいい (わかりやすい)

Reply:
修正します！